spec.bs

```diff
@@ -2280,6 +2268,27 @@ To mend the intended distinction between [=top-level traversables=] and [=fenced
 Note: With these new definitions, a [=top-level traversable=] is essentially "unfenced" as described
 in the [[#nested-traversables-intro]].
 
+The algorithm as it exists today is a "fenced" algorithm, which will be re-introduced as a "fenced"
+variant as follows:
+
+<div algorithm>
+  A <dfn>fenced top-level traversable</dfn> is a [=traversable navigable=] with a null
```

Comment:
Like the below, I'm not sure this is necessary.

Reply:
I think this was referring to the `noexport` comment? I'm going to resolve this but please re-open if this is referring to something else.